ops/testing.py

```diff
@@ -505,6 +505,16 @@ def get_container_pebble_plan(
             raise KeyError('no known pebble client for container "{}"'.format(container_name))
         return client.get_plan()
 
+    def container_pebble_ready(self, container_name: str):
+        """Fire the pebble_ready hook for the associated container.
+
+        This will do nothing if the begin() has not been called.
```

Comment:
Should it do nothing or raise an error since it seems like a mistake to attempt to fire a ready event prematurely?